apps/portals/src/configurations/elportal/synapseConfigs/studies.ts

```diff
@@ -109,11 +114,11 @@ export const studiesDetailsPageProps: DetailsPageProps = {
         },
         {
           name: 'CardContainerLogic',
-          columnName: 'studyKey',
+          columnName: 'relatedStudies',
           title: 'Related Studies',
-          tableSqlKeys: ['relatedStudies'],
+          tableSqlKeys: ['id'],
           props: {
-            sqlOperator: ColumnMultiValueFunction.HAS,
+            sqlOperator: ColumnSingleValueFilterOperator.EQUAL,
```

Comment:
I think this was backwards. Use the current row's `relatedStudies` column to filter `id`s in the studies table

Reply:
Oops!  Did this always return zero results?

Reply:
I think it worked before the latest schema changes since it just looked up the same relationship in reverse (...assuming Study X is related to Y also implies Y is related to X)